packages/react/package.json

```diff
@@ -32,7 +32,8 @@
     "graphql": "16.5.0",
     "lodash": "^4.17.21",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "typescript": "5.0.4"
```

Comment:
Added `typescript` as a dependency to this package only because we need to keep the root typescript pinned to `4.7.4` to keep prettier happy. Prettier does work if you recreate `yarn.lock` but that breaks the build so this is the easiest way to keep it working